### ISSUE TITLE
new, backup meta info

### ISSUE THREAD

**Issue by wnarutou** [LeslieLeung/reaper#38]
Is it possible to add a feature that backs up all the meta info in the repository, such as issues/wiki/prs.

**Comment by LeslieLeung** [comment]
There are APIs for:
- issues: https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#list-repository-issues
- wiki: https://docs.github.com/en/repositories/archiving-a-github-repository/backing-up-a-repository#backing-up-a-wiki-with-the-git-cli
- prs: https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#list-pull-requests

So backing up issues, wiki and prs would be possible. But with what format do you want to save them? I think simple raw json should do the trick.

**Comment by wnarutou** [comment]
> There are APIs for:
> 
> * issues: https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#list-repository-issues
> * wiki: https://docs.github.com/en/repositories/archiving-a-github-repository/backing-up-a-repository#backing-up-a-wiki-with-the-git-cli
> * prs: https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#list-pull-requests
> 
> So backing up issues, wiki and prs would be possible. But with what format do you want to save them? I think simple raw json should do the trick.

yeah, json is awesome, If you could do that, I would be so grateful.
